test_utils/factories.py

```diff
@@ -610,3 +611,41 @@ class Meta:
 
     user_id = factory.LazyAttribute(lambda x: FAKER.pyint())
     course_id = factory.LazyAttribute(lambda x: FAKER.slug())
+
+
+class CanvasGlobalConfigurationFactory(factory.django.DjangoModelFactory):
+    """
+    ``CanvasGlobalConfiguration`` factory.
+
+    Creates an instance of ``CanvasGlobalConfiguration`` with minimal boilerplate.
+    """
+
+    class Meta:
+        """
+        Meta for ``CanvasGlobalConfiguration``.
+        """
+
+        model = CanvasGlobalConfiguration
+
+    oauth_api_path = factory.LazyAttribute(lambda x: FAKER.file_path())
+    course_api_path = factory.LazyAttribute(lambda x: FAKER.file_path())
```

Comment:
I would check with @alex-sheehan-edx if this field is needed still? I thought he moved it to appconfig

Reply:
Yeah this will likely need rebasing off of master anyways. I just had it in there at the time because that PR had not merged.

Reply:
yeah course_api_path is still there but this oauth_api_path is unnecessary